### ISSUE TITLE
add translated user manual

### ISSUE THREAD

**Issue by dacoex** [nextcloud/documentation#768]
please add a link to the translated German user manual to:
https://docs.nextcloud.com/

Forum reference:
https://help.nextcloud.com/t/nextcloud-user-manual-guide/12287

**Comment by MorrisJobke** [comment]
cc @skjnldsv

**Comment by tflidd** [comment]
There were also efforts for a French manual (https://fr.wikibooks.org/wiki/Manuel_utilisateur_Nextcloud) and a more general topic to pass all translations of the user manual to transifex: : https://github.com/nextcloud/documentation/issues/216

**Comment by dacoex** [comment]
Yes, it would be great to have these translated manuals linked on the main docs page.
No more search engine...

**Comment by wdfee** [comment]
...and linked to the translated manual at the Help menu item in the user menu (top right of the Nextcloud Web interface) in user's language if available.

**Comment by pierreozoux** [comment]
I opened the issue on the read the doc theme:

https://github.com/readthedocs/sphinx_rtd_theme/issues/857

If anybody is fluent in this, I'd gladly learn from you, and implement it, but currently, I'm afraid it is impossible with the open source version of the sphinx theme.

**Comment by joshtrichards** [comment]
This was done awhile ago. And recently made more prominent (for #8310) via #13404
